python/ray/tests/test_basic.py

```diff
@@ -1206,6 +1206,25 @@ def f(self, x):
     assert ray.get(obj_id) == 2
 
 
+def test_direct_actor_large_objects(ray_start_regular):
+    @ray.remote
+    class Actor(object):
+        def __init__(self):
+            pass
+
+        def f(self):
+            time.sleep(1)
+            return np.zeros(10000000)
```

Comment:
I don't think this actually returns an array of this size.
```
In [9]: @ray.remote
   ...: def f():
   ...:     return np.zeros(10000000)
   ...:

In [10]: x = f.remote()

In [11]: y = ray.get(x)

In [12]: import sys

In [13]: sys.getsizeof(y)
Out[13]: 96
```

Reply:
It does work, getsizeof just doesn't work on numpy arrays:

>>> np.zeros(2222)
array([0., 0., 0., ..., 0., 0., 0.])

Reply:
Also doesn't work on pickled arrays:
```
s = pickle.dumps(np.zeros(10000000))
sys.getsizeof(pickle.loads(s)) 
```

also returns 96, so maybe it is ok...